automated-comments/ruby/general/explicit_return.md

```diff
@@ -0,0 +1,13 @@
+Remove the explict return to simplify the method.
+
+In Ruby, a method will return the object that was returned from the last evaluated statement. Explicitly using the `return` keyword is therefore unnecessary unless we are returning early in a method.
```

Comment:
```suggestion
In Ruby, a method will return the result of the last evaluated statement. Explicitly using the `return` keyword is therefore unnecessary, unless when returning early in a method.
```

A bit more concise and without personas?

Reply:
Updated `unless` for `except`